andela/k2-cfh
# feature(czar-should-be-able-to-draw-a-card): add black card for czar …

#### What does this PR do?

Re-Design UI for the gaming screen

#### Description of Task to be completed?

- CZARs should be able to draw cards by clicking

#### How should this be manually tested?
- Clone the repository
- Run `git checkout feature/153009263/czar-should-be-able-to-draw-a-card`
- Run `npm start` to start server
- Click the start playing button to see the new czar card modal

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: feature
- story id: 153009263
- story title: CZAR should be able to draw cards by clicking

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2017-12-17 at 6 06 14 pm" src="https://user-images.githubusercontent.com/28455813/34081907-74c3cff8-e355-11e7-8699-0ee85b51b521.png">


#### Questions:

N/A
